publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java

```diff
@@ -30,12 +33,20 @@ public BasePublishHookTest() throws Exception {
         super();
     }
 
+    @Override
+    public Set<String> getHooksToRemove() {
+        return new HashSet<>();
+    }
+    
     @Before
     public void before() throws UnknownHostException {
         cleanupMongoCollections("country", "esbEvents");
     }
 
     protected void insertCountry() throws Exception {
+        com.mongodb.DBCursor cursor=mongoServer.getConnection().getDB("testdb").getCollection("metadata").find();
+        while(cursor.hasNext())
+            System.out.println(cursor.next());
```

Comment:
Printing metadata in the DB. That's how I figured out hooks are being
removed: there are no hooks in the metadata in the db.

On Tue, Sep 22, 2015 at 9:15 AM, Dennis Crissman <notifications@github.com>
wrote:

> In
> publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java
> <https://github.com/lightblue-platform/lightblue-esb-hook/pull/33#discussion_r40098904>
> :
>
> >      @Before
> >      public void before() throws UnknownHostException {
> >          cleanupMongoCollections("country", "esbEvents");
> >      }
> >
> >      protected void insertCountry() throws Exception {
> > +        com.mongodb.DBCursor cursor=mongoServer.getConnection().getDB("testdb").getCollection("metadata").find();
> > +        while(cursor.hasNext())
> > +            System.out.println(cursor.next());
>
> I don't understand what this is doing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-esb-hook/pull/33/files#r40098904>
> .
>